ext/handlers_curl.c

```diff
@@ -176,7 +176,7 @@ static HashTable *ddtrace_curl_multi_get_gc(zend_object *object, zval **table, i
 }
 
 static pthread_once_t dd_replace_curl_get_gc_once = PTHREAD_ONCE_INIT;
-static zend_object_handlers *dd_curl_object_handlers;
+ZEND_TLS zend_object_handlers *dd_curl_object_handlers = NULL;
```

Comment:
I don't understand this change?

Reply:
Previously on every call to `curl_multi_init`, every thread would write to this address. This is obviously not thread-safe. Making it thread-local fixes the problem and required no further changes, because whichever thread wins the Once control will write to handler using its thread-local handlers. Does that make sense?

Reply:
Yes, every thread would write to that address, but it would be the same value over and over again, making it unproblematic.
At least we assume that the curl handlers are the same for each curl_multi_handle, otherwise the whole thing doesn't work anyway.

Reply:
But it doesn't do harm to change it - so I don't care.